docs/open-source-project/release-notes/tizen-5-0-m1.md

```diff
@@ -0,0 +1,952 @@
+
+​		**o**   **Pulseaudio 5.0 -> 11.1**
+
+ 
+
+## **Network and Connectivity**
+
+ **New and Changed Features**
+
+​	**·**        **Network Monitoring (In-house only module)**
+
+​		**o**   **Support for ethernet cable state feature has been added**
+
+​		**o**   **Support for Wi-Fi module state feature has been added**
+
+​		**o**   **Supported for IP conflict detection feature has been added**
+
+​		**o**   **Supported for tcpdump feature has been added**
+
+​		**o**   **Supported for statistics feature has been added**
+
+​		**o**   **Supported for TCP congestion feature has been added**
+
+​		**o**   **Supported for TCP Tx retry rate feature has been added**
+
+​		**o**   **Supported for cellular state feature has been added**
+
+​		**o**   **Supported for Wi-Fi state feature has been added**
+
+​		**o**   **Supported for ethernet state feature has been added**
+
+​		**o**   **Supported for scan state feature has been added**
+
+​		**o**   **Supported for default connection fuction has been added**
+
+​		**o**   **Supported for getting the connection related information (e.g,. proxy address, prefix length, IP address) functions have been added**
+
+​		**o**   **Supported for getting the Wi-Fi related informations (e.g,. BSSID, ESSID, RSSI level, frequency) functions have been added**
+
+​	**·**        **Network Firewall**
+
+​		**o**   **Supported for In-bound/out-bound rule management has been added**
+
+​		**o**   **Supported for IP, port and protocol type control has been added**
+
+​		**o**   **Supported for Accept, Drop and Log targets has been added**
+
+​		**o**   **Supported for IPv4 and IPv6 rules has been added**
+
+​		**o**   **Supported for rule chain management has been added**
+
+​		**o**   **Supported for Tunneling functions has been added**
+
+​		**o**   **Supported for lock/unlock functions has been added**
+
+​	**·**        **Multipath TCP (In-house only module)**
+
+​		**o**   **New feature for multipath TCP support has been added**
+
+​			**§**  **A connection API (In-house only) for enabling the multipath TCP function has been added**
+
+​		**§**  **A connection API (In-house only) for setting the path-manager for multipath TCP has been added**
+
+​			**§**  **A connection API (In-house only) for setting the scheduling for multipath TCP has been added**
+
+​	**·**        **TCP Fast Open**
+
+​		**o**   **A http API for allowing TCP Fast Open has been added**
+
+​	**·**        **SoftAP**
+
+​		**o**   **capi-network-softap has been open to the public**
+
+​	**·**        **MTP FunctionFS**
+
+​		**o**   **FunctionFS has been added in USB File System which is based on MTP operation**
+
+​	**·**        **Wi-Fi Native CAPI**
+
+​		**o**   **capi-wifi has been removed and instead, capi-wifi-manager has been replaced**
+
+​		**o**   **New features and APIs** in mobile, wearable, and TV profiles**:**
+
+​			§  **A wifi-manager** API for getting the all VSIE(**Vendor Specific Information Elements)** of AP has been added.

```

Comment:
Added a **wifi-manager** API for getting all the VSIE(**Vendor Specific Information Elements)** of AP

This comment is global for all the points..

Reply:
will accept.